bot/utilities/custom/player.py

```diff
@@ -170,7 +172,23 @@ async def handle_track_end(self, *, error: bool = False) -> None:
             return
 
         self._waiting = True
-        track = await self.queue.get_wait()
+        track = None
```

Comment:
i think this is unessacery? `Queue.get_wait` *should* always return a track and the timeout will return anyway